packages/ckeditor5-dev-env/tests/translations/download.js

```diff
@@ -63,66 +92,44 @@ describe( 'download', () => {
 			}
 		};
 
-		const translations = {
+		translations = {
 			'ckeditor5-core': {
-				pl: {
-					content: 'ckeditor5-core-pl-content'
-				}
+				pl: { content: 'ckeditor5-core-pl-content' }
 			},
 			'ckeditor5-ui': {
-				en_AU: {
-					content: 'ckeditor5-ui-en-content'
-				}
+				en_AU: { content: 'ckeditor5-ui-en-content' }
 			}
 		};
 
-		const fileContents = {
+		fileContents = {
 			'ckeditor5-core-pl-content': {}, // empty
```

Comment:
`// empty` :D 

We see that the object is empty. If you didn't explain why this object is empty, I suggest removing this comment.